### PR TITLE
Add Coinomi Wallet 1.2.4

### PR DESCRIPTION
**Pull request by danielbayley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by danielbayley** [comment]
No idea why this is failing…

File: Casks/coinomi-wallet.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "coinomi-wallet" do
+  version "1.2.4"
+  sha256 "da8de4e58b4de69454449e4aad2f69daab658c40c6669d8b2e2d434b26c5dcf4"
+
+  url "https://storage.coinomi.com/binaries/desktop/coinomi-wallet-#{version}-macos.dmg",
+      user_agent: :fake
+  appcast "https://coinomi.com/downloads"
```
Comment (vitorgalvao) [inline Casks/coinomi-wallet.rb:7]: Please convert to a [`livecheck`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/livecheck.md).

**Review by vitorgalvao — DISMISSED** [review]

File: Casks/coinomi-wallet.rb (hunk 1 above)
Comment (danielbayley) [reply Casks/coinomi-wallet.rb]: > Please convert to a [`livecheck`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/livecheck.md).

@vitorgalvao Interesting… I missed this development. Does this break `brew audit --appcast`? Is there an equivalent test like `brew audit --livecheck` or something?

Also, there seems to be `strategy`s missing from that documentation… Namely `:git`, `:github_latest`, `:electron_builder` that I have come across.

**Comment by danielbayley** [comment]
@vitorgalvao Looks like the `livecheck` `regex()` is failing on the raw HTML fetched with `:page_match` (`curl`?)… Should something like `CGI.unescapeHTML` be applied inside that method?

**Comment by danielbayley** [comment]
> Please convert to a [`livecheck`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/livecheck.md).

@vitorgalvao Interesting… I missed this development. Does this break `brew audit --appcast`? Is there an equivalent test like `brew audit --livecheck` or something?

Also, there seems to be `strategy`s missing from that documentation… Namely `:git`, `:github_latest`, `:electron_builder` that I have come across.

**Comment by miccal** [comment]
Thank you @danielbayley.

**Review by miccal — APPROVED** [review]

**Comment by vitorgalvao** [comment]
@danielbayley `livecheck` already existed for formulae and I’m not intimately familiar with the inner workings. You can test it with `brew livecheck --debug CASK_NAME`.

There’s also a [`livecheck` document in the core’s repo](https://github.com/Homebrew/brew/blob/master/docs/Brew-Livecheck.md) which [links to the strategies](https://rubydoc.brew.sh/Homebrew/Livecheck/Strategy.html). But I wouldn’t be opposed to making them clearer in the markdown documentation.

**Pull request merged**